background.ts

```diff
@@ -524,7 +524,7 @@ ipcMainProxy.on('api-get-credentials', () => {
 
 Electron.ipcMain.handle('api-get-credentials', () => {
   return new Promise<void>((resolve) => {
-    mainEvents.once('api-credentials', resolve);
+    mainEvents.once('api-credentials', () => resolve());
```

Comment:
Something about this change causes `fetchCredentials` to return `undefined`

https://github.com/rancher-sandbox/rancher-desktop/blob/50188b15e989abb43ee822e73647a0975e69d71b/pkg/rancher-desktop/store/credentials.ts#L31-L36